target/linux/ath79/dts/qca9561_tplink_archer-c25-v1.dts

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,archer-c25-v1", "qca,qca9560";
```

Comment:
Oh boy, that "qca,qca9560" should be "qca,qca9561" since there is no "qca9560". But thing is: this seems to wrong for all the currently existing qca9561 DTS targets as well. But almost all qca9563 targets (apart from the rosinson_wr818.dts - which I'm going to fix reaal quick) are using "qca,qca9563". Oh well.

Note: The QCA956x/0 is a bit confusing. It's because according to https://wikidevi.com/wiki/Qualcomm_Atheros, there's the qca9561, qca9563 and TP9343(which is a cut-down QCA9563 for TP-Link). As well as QCA9565 but this one is a wlan+bt pcie Chip and not a SoC.

Reply:
Indeed. I just used "qca,qca9560" because it was in qca9561_tplink_archer-c5x.dtsi and I thought someone might have used it for a reason.
Will change.